### PR TITLE
28/global state

### PR DESCRIPTION
**Pull request by hagoptaminian**

### CODE REVIEW

**Comment by hagoptaminian** [comment]
@pyphilia: these are the minor changes in #67 and #68. For #28, I basically moved the app-specific state to a separate reducer `lab.js`, except for `stageDimensions`, which I kept in `layout.js`. I double-checked all the `mapStateToProps` and `useSelector`s to make sure they are getting state from the right place, and ran the app and it's working correctly.

File: src/reducers/layout.js [hunk 1]
```diff
@@ -38,25 +13,6 @@ const INITIAL_STATE = {
   showSideMenu: true,
   lab: {
```
Comment (pyphilia) [inline src/reducers/layout.js:14]: Is it necessary to have this structure lab.stageDimensions? Maybe we can get rid of `lab`.

**Review by pyphilia — APPROVED** [review]
Really nice 🦜  I just left a minor suggestion.

File: src/reducers/layout.js (hunk 1 above)
Comment (hagoptaminian) [reply src/reducers/layout.js:14]: Good idea 👍

**Pull request merged**